### DISCUSSION TITLE
A way to clear all populated fields in request, options, response to start a new call with a clean slate

### DISCUSSION THREAD

**Discussion by ozsoft** [hoppscotch/hoppscotch#1757] [Ideas]
It will really help when using the Hoppscotch to be able to clear everything to start a new call, just like you would with other applications by creating a new tab. I know you can call the current call and that is added to your history but I think having the ability to clear everything does help when making lots of calls.

**Comment by liyasthomas** [comment]
Click on the "Clear all" icon button to reset the request state to default.

Screenshot:

<img width="1440" alt="Screenshot 2021-08-04 at 4 19 02 PM" src="https://user-images.githubusercontent.com/10395817/128168933-fcba81df-cd21-405a-8b90-f76d47824421.png">
